File: AVeryBigSum/AVeryBigSum.py
# Transpile AVeryBigSum.js to Python

# given an array of integers find sum of integers

# create a function called aVeryBigSum
def aVeryBigSum(arr):
  # create a var to hold the sum
  sum = 0
  # iterate array of integers
  for int in arr:
    # add each integer to sum var
    sum += int
  # return sum var
  return sum

# TESTS
print(aVeryBigSum([1000000001, 1000000002, 1000000003, 1000000004, 1000000005]))
# -> 5000000015

File: AVeryBigSum/AVeryBigSum.js
/* SUMMARY
Return the sum of large ints in an ar.
*/

/* OBJECTIVES
-> aVeryBigSum should take in largeNumbers
-> largeNumbers should be an arr of ints
-> aVeryBigSum should return the sum of the ints in largeNumbers
 */

/* CONSTRAINTS
-> largeNumbers will have no more than 10 or no less than 1 ints
-> each int in largeNumbers will be no less than 0 or more than 10000000000
 */

// SOLUTIONS
const aVeryBigSum = (largeNumbers) => {
  let counter = 0;
  for (let idx = 0; idx < largeNumbers.length; idx++) {
    counter += largeNumbers[idx];
  }
  return counter;
};

// TESTS
console.log(aVeryBigSum([1000000001, 1000000002, 1000000003, 1000000004, 1000000005]));
// -> 5000000015
